MIGRATION.md

```diff
@@ -0,0 +1,67 @@
+# Migrating from JSONForms 1 (AngluarJS)
+The complexity of the migration of an existing JSONForms 1 application, which is based on AngularJS, to JSONForms 2 depends on the feature set you use.
+
+## Main Features of JSONForms 2
+There are two big changes between JSONForms 1 and JSONForms 2 you need to understand when migrating your existing application.
+
+1. JSONForms 2 uses [redux](https://redux.js.org/) in order to maintain the state. Your application will need to add redux as a dependency.
+
+2. JSONForms 2 does not rely on any specific ui framework. The `2.0.0` release only provides renderers based on [React](https://reactjs.org) but we are working on an [Angular](https://angular.io) renderer-set for `2.1.0`.
```

Comment:
...on any specific UI framework [or library].
Rephrase 2nd sentence: The 2.0.0 initial release features renderers based on React. A Angular based renderer set will be be released with 2.1.0